lib/plugins/aws/invokeLocal/resolvers/sqs.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+const aws = require('aws-sdk');
+const resolvers = require('.');
+
+module.exports = {
+  'Ref': resolvers.resolveResourceId,
+  'Fn::GetAtt': (plugin, resource, resourceName, parameters, serviceConfig) => {
+    const sqs = new aws.SQS(serviceConfig);
+    switch (parameters[1]) {
+      case 'Arn':
+        return resolvers.getCFResources(serviceConfig).then(cfData => {
+          const cfResource = cfData.find(x => x.LogicalResourceId === resourceName);
+          if (!cfResource) {
+            return parameters;
```

Comment:
I think we should crash here with message that as Queue was not created yet we cannot resolve it's ARN.

At least returning an object where a string env var is expected introduces a buggy behavior.